yt_dlp/extractor/francetv.py

```diff
@@ -191,19 +213,19 @@ def _extract_video(self, video_id, catalogue=None):
                 } for sheet in spritesheets]
             })
 
-        if subtitle:
-            title += ' - %s' % subtitle
-        title = title.strip()
-
         return {
             'id': video_id,
-            'title': title,
+            'title': (title + ' - %s' % subtitle if subtitle else title).strip(),
```

Comment:
```suggestion
            'title': join_nonempty(title, subtitle, delim=' - ').strip(),
```
Import `join_nonempty` from `..utils`